src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,114 @@
+
+The JSON-RPC server is encouraged to accept at least one `archive_unstable_storageDiff` subscription per JSON-RPC client. Trying to open more might lead to a JSON-RPC error when calling `archive_unstable_storageDiff`. The JSON-RPC server must return an error code if the server is overloaded and cannot accept new subscriptions.
+
+## Notifications format
+
+This function will later generate notifications with the following format:
+
+```json
+{
+    "jsonrpc": "2.0",
+    "method": "archive_unstable_storageDiffEvent",
+    "params": {
+        "subscription": "...",
+        "result": ...
+    }
+}
+```
+
+Where `subscription` is the value returned by this function, and `result` can be one of the following:
+
+### differences
+
+```json
+{
+    "event": "differences",
+    "differences": [
+        {
+            "key": "0x...",
+            "value": "0x...",
+            "hash": "0x...",
+            "type": "added" | "modified" | "deleted",
+        },
+    ],
+}
+```
+
+The `differences` field contains an array of objects, each describing a storage difference.
+
+The `key` field contains the hexadecimal-encoded key of the storage entry. If the a key prefix was provided in the `items` parameter, the `key` field is guaranteed to start with one of the key prefixes provided. If the `excludeKeyPrefixes` field was provided in the `items` parameter, the `key` field is guaranteed not to start with any of the excluded key prefixes provided.
+
+The `value` field contains the hexadecimal-encoded value of the storage entry. This field is present when prefixes are not provided or when the `type` field in the `items` parameter is set to `value`.
+
+The `hash` field contains the hexadecimal-encoded hash of the storage entry. This field is only present when the `type` field in the `items` parameter is set to `hash`.
+
+When the `type` field in the `items` parameter is set to `none`, the `value` and `hash` fields are not present.
+
+The `type` field indicates the type of the storage difference. The possible values are:
+
+- `added`: The storage entry was added.
+- `modified`: The storage entry was modified.
+- `deleted`: The storage entry was deleted.
+
+### continue
+
+```json
+{
+    "event": "continue",
+}
+```
+
+This notification is sent when the storage difference calculation is paused and needs to be resumed. The user must call `archive_unstable_storageDiffContinue` with the subscription value to continue the calculation.
```

Comment:
`archive_unstable_continue` would more closely align with `chainHead_v1_continue`, unless there's a reason to be more specific here?

Reply:
Made the subscription a method after all, thanks